BitwardenShared/Core/Autofill/Utilities/Fido2DebugginReportBuilder.swift

```diff
@@ -0,0 +1,49 @@
+import BitwardenSdk
+import Foundation
+
+/// Report with traceability about Fido2 flows.
+public struct Fido2DebuggingReport {
```

Comment:
🎨 Since we only use these objects in Debug builds, should we gate all of the definitions behind `#if DEBUG` as well?